ooni/deck.py

```diff
@@ -85,11 +88,30 @@ def nettest_to_path(path, allow_arbitrary_paths=False):
     if allow_arbitrary_paths and os.path.exists(path):
         return path
 
-    fp = FilePath(config.nettest_directory).preauthChild(path + '.py')
-    if fp.exists():
-        return fp.path
-    else:
+    p = path.split("/")
```

Comment:
Fair point. I guess we could do an `rsplit` and get the last item as the test_name and ignore the test_category section.